components/dropdown/dropdown.js

```diff
@@ -22,6 +22,11 @@ class Dropdown extends DropdownOpenerMixin(LitElement) {
 	 */
 	getOpenerElement() {
 		if (!this.shadowRoot) return undefined;
+		if (this.slottedOpener)  {
+			return this.querySelector('slot[name="opener"]')
```

Comment:
Is this the type of scenario you are wanting to support?
```html
my-component
   --shadowDOM
      <d2l-dropdown>
         <slot></slot>
         <d2l-dropdown-content max-width="400">
            ...
         </d2l-dropdown-content>
      </d2l-dropdown>
```

```html
<my-component>
   <d2l-button class="d2l-dropdown-opener">Open it!</d2l-button>
</my-component>
```

If so, can we just add the `{ flatten: true }` to the original code?  The `flatten` option will resolve the slots.

Reply:
Hmm.. that doesn't seem to be working for me. It can't find the default slot in that case.